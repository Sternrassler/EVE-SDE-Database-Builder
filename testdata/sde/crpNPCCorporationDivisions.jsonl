{"divisionID":1,"divisionName":"Accounting","description":"Financial management division","leaderType":"CFO"}
{"divisionID":2,"divisionName":"Security","description":"Corporate security division","leaderType":"CSO"}
{"divisionID":3,"divisionName":"Production","description":"Manufacturing and production division","leaderType":"CPO"}
