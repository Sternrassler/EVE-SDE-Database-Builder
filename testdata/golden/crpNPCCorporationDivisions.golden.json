[
  {
    "corporationID": 0,
    "divisionID": 1,
    "size": null,
    "divisionName": "Accounting",
    "leaderID": null
  },
  {
    "corporationID": 0,
    "divisionID": 2,
    "size": null,
    "divisionName": "Security",
    "leaderID": null
  },
  {
    "corporationID": 0,
    "divisionID": 3,
    "size": null,
    "divisionName": "Production",
    "leaderID": null
  }
]